### ISSUE TITLE
Error: invalid memory address or nil pointer dereference 

### ISSUE THREAD

**Issue by spaluchowski** [americanexpress/earlybird#28]
I have just built the binaries from the source code (both linux/amd64 and windows/amd64 behave the same way).
When executing it I get as a result:

> panic: runtime error: invalid memory address or nil pointer dereference
> [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xa12032]
> 
> goroutine 1 [running]:
> github.com/americanexpress/earlybird/pkg/core.(*EarlybirdCfg).GetRuleModulesMap.func1(0xc000026f60, 0x19, 0x0, 0x0, 0xc47f00, 0xc00010ed80, 0xc000107be8, 0x4108ad)
>         /var/lib/jenkins/workspace/Earlybird-build-binaries/earlybird/pkg/core/core.go:148 +0x32
> path/filepath.Walk(0xc000026f60, 0x19, 0xc000107c30, 0xc000026f60, 0x19)
>         /usr/lib/golang/src/path/filepath/path.go:404 +0x6b
> github.com/americanexpress/earlybird/pkg/core.(*EarlybirdCfg).GetRuleModulesMap(0x1044e80, 0x10445e0, 0xc00002c810)
>         /var/lib/jenkins/workspace/Earlybird-build-binaries/earlybird/pkg/core/core.go:147 +0xef
> github.com/americanexpress/earlybird/pkg/core.(*EarlybirdCfg).ConfigInit(0x1044e80)
>         /var/lib/jenkins/workspace/Earlybird-build-binaries/earlybird/pkg/core/core.go:176 +0x2ef
> main.main()
>         /var/lib/jenkins/workspace/Earlybird-build-binaries/earlybird/go-earlybird.go:47 +0x35d

**Comment by grinish21** [comment]
Can you share the steps you followed??

**Comment by spaluchowski** [comment]
git clone https://github.com/americanexpress/earlybird.git
cd earlybird/
export GOOS=windows
export GOARCH=amd64
go build -o binaries/go-earlybird.exe
cd binaries
go-earlybird.exe

and the panic runtime error shows

**Comment by pmartindev** [comment]
@spaluchowski I was running into the same issues, and noticed that my config/ files were not in the same directory as the binary. Moving those to the directory seemed to solve the issue for me 👍

**Comment by grinish21** [comment]
the issue seems to be related to the config files folder structure. Closing the issue due to no response.

**Comment by juju4** [comment]
I got this issue too and yes, it's earlybird config related.
on my side, I solved it by copying manually config/* to ~/.go-earlybird/

but message is misleading (not memory related at all) and IMHO, it should be made clearer than some files are missing and earlybird could not set them up on its own.
https://github.com/americanexpress/earlybird/blob/main/pkg/core/core.go#L148

Thanks for your work and sharing the tool!
